### PR TITLE
Update README examples to 09-06

### PR DESCRIPTION
**Pull request by bosr**
Following @Danappelxx suggestion, here are the updated docs. I heading to the GitBook now :)

As a side note, I updated `.swift-version` to 09-06.

Cheers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1637,122 +1634,147 @@ func ==(lhs: Item, rhs: Item) -> Bool {
     return lhs.GUID == rhs.GUID
 }
 
+enum Result<T> {
```
Comment (Danappelxx) [inline README.md]: Maybe we could reuse `ChannelResult` here?

File: README.md [hunk 2]
```diff
@@ -1637,122 +1634,147 @@ func ==(lhs: Item, rhs: Item) -> Bool {
...
+	case value(T)
+	case failure(Error)
+
+	func success(completionHandler: (T) -> ()) {
+		switch self {
```
Comment (Danappelxx) [inline README.md]: Could you change the indentation in these snippets from tabs to 4 spaces in this snippet?

File: README.md [hunk 3]
```diff
@@ -1637,122 +1634,147 @@ func ==(lhs: Item, rhs: Item) -> Bool {
...
+			struct LocalError : Error, CustomStringConvertible { let description: String }
+			return Result.failure(LocalError(description: "Network Error"))
+		}
+	}
 }
 
 protocol SubscriptionType {
-    var updates: ReceivingChannel<Item> { get }
-    func close() -> ErrorProtocol?
+	var uprint_headerates: ReceivingChannel<Item> { get }
```
Comment (Danappelxx) [inline README.md]: `uprint_headerates` - A typo, maybe? 😝

**Comment by Danappelxx** [comment]
Left a couple comments. Other than that, awesome job!

File: README.md (hunk 2 above)
Comment (bosr) [inline README.md]: Yes, I personally use 4 spaces: how did those become tabs? -> will fix

File: README.md (hunk 1 above)
Comment (bosr) [inline README.md]: I had figured out you had something like this but could not find it. I will fix this

File: README.md (hunk 3 above)
Comment (bosr) [inline README.md]: 💯

**Comment by bosr** [comment]
I checked it also passes on 09-11-a (not much of a surprise, though)

File: README.md (hunk 3 above)
Comment (Danappelxx) [inline README.md]: Looks to me like it's still there? I was referring to `upring_headerates` to `updates`.

Comment (bosr) [inline README.md]: fixed!

**Comment by bosr** [comment]
Thanks for the fine corrections @Danappelxx

**Comment by Danappelxx** [comment]
Wonderful!

**Pull request merged**